src/components/CTASecondary.js

```diff
@@ -29,35 +28,34 @@ const CTASecondary = ({ classes, className }) => (
 export default withStyles((theme) => ({
   root: {
     display: "flex",
-    justifyContent: "center",
     backgroundColor: "#55C3BA",
     padding: "40px 12px",
-
-    [theme.breakpoints[900]]: {
-      padding: "60px 12px",
-    },
+    justifyContent: "center",
+    color: "#FFF",
   },
-  ctaContainer: {
+  inner: {
     display: "flex",
+    width: "100%",
+    justifyContent: "center",
```

Comment:
This is an extreme nitpick– I try to group similar CSS attributes together.

So in this case, I would expect `display: flex, flexDirection, and justifyContent` to all be next to each other.

Same thing with width/maxWidth, I'd expect those attributes to be next to each other.

Reply:
I like it! Makes sense to me 👍